core-feature-pack/src/main/resources/content/bin/systemd/README

```diff
@@ -0,0 +1,26 @@
+= How to configure WildFly as a systemd service
+
+== Create a wildfly user
+
+    # groupadd -r wildfly
+    # useradd -r -g wildfly -d /usr/share/wildfly -s /sbin/nologin wildfly
+
+== Install WildFly
+
+    # mkdir /usr/share/wildfly/
```

Comment:
Installation should probably be in /opt.  Data files should be (I guess) in /var/opt.  I would at least use http://www.linuxbase.org/betaspecs/fhs/fhs.html as a reference.